sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosClientBuilder.java

```diff
@@ -377,6 +378,36 @@ public CosmosClientBuilder contentResponseOnWriteEnabled(boolean contentResponse
         return this;
     }
 
+    /**
```

Comment:
Please add support to enable this per request - see https://github.com/Azure/azure-cosmos-dotnet-v3/issues/1496 and https://github.com/Azure/azure-cosmos-dotnet-v3/pull/1565

Reply:
I'd wait until this goes in .NET?

Reply:
Not blocking - IMO when we touch the code we should just fix it. We are targeting to ensure that any new feature will be "released" only when it is available in Java SDK - so don't think there is a hard order that it needs to go into .Net first. But again - not blocking.

Reply:
I have few more changes lines up, mostly patch and batch-patch. I can put those changes there and till the time .Net will also be finalized/clear.